web/source/dom/uiTouchHandlerBase.ts

```diff
@@ -249,6 +285,9 @@ namespace com.keyman.dom {
       var x = e.changedTouches[0].pageX;
       var beyondEdge = ((x < 2 && this.touchX > 5) || (x > window.innerWidth - 2 && this.touchX < window.innerWidth - 5));
 
+      // Allow an accidental fudge-factor for overflow element noise, but not much.
+      beyondEdge = beyondEdge || this.scrollTouchState.totalLength > 10;
```

Comment:
Please make the `10` into a constant; I know we've got older code that doesn't religiously use constants, but I'd like to try and avoid perpetuating this :)

Reply:
Seconding this! Please make it a constant like... `EDGE_FUDGE_FACTOR`...?